databases/week1/create_table.sql

```diff
@@ -0,0 +1,55 @@
+-- Active: 1660473813574@@0.0.0.0@3306@hyf_lesson_1
+
+--1. Find out how many tasks are in the task table
+SELECT COUNT(id) 
+FROM task;
+
+--2. Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(id) 
+FROM task 
+WHERE due_date IS NULL;
+
+--3. Find all the tasks that are marked as done
+SELECT task.title AS TITLE, status.name AS STATUS 
+FROM task 
+JOIN status ON task.status_id = status.id
+WHERE status.name = 'Done';
+
+--4 Find all the tasks that are not marked as done
+SELECT task.title AS TITLE, status.name AS STATUS 
+FROM task 
+JOIN status ON task.status_id = status.id
+WHERE status.name <> 'Done';
```

Comment:
ah cool, I didn't know about this operator <> https://dba.stackexchange.com/questions/155650/authoritative-source-that-and-are-identical-in-performance-in-sql-server